tests/data/collected_variables.json

```diff
@@ -0,0 +1 @@
+{"temp_djf_iamean_s0p_hist": 0.8602696061134338}
```

Comment:
While it is good to keep our tests simple we may want to consider adding a couple more examples in here to ensure the tests cover other cases.